apps/server/src/modules/learnroom/common-cartridge/common-cartridge-file-builder.ts

```diff
@@ -26,14 +26,15 @@ export interface ICommonCartridgeOrganizationBuilder {
 
 export interface ICommonCartridgeFileBuilder {
 	addOrganization(props: ICommonCartridgeOrganizationProps): ICommonCartridgeOrganizationBuilder;
+
 	addResourceToFile(props: ICommonCartridgeResourceProps): ICommonCartridgeFileBuilder;
+
 	build(): Promise<Buffer>;
 }
 
 class CommonCartridgeOrganizationBuilder implements ICommonCartridgeOrganizationBuilder {
```

Comment:
I think Putting this class in a separate class would improve the readability of the code.

Reply:
I tried to achive information hiding. If I put this class into another file and export it, than anyone can access this class.